updater/reports/ReportReposPersonalNonOwnerPush.py

```diff
@@ -0,0 +1,38 @@
+from .ReportDaily import *
+
+# Find personal repositories that non-owners are pushing to.
+# These repositories should be moved into organizations.
+# Only look at active users (not suspended!) and only look at pushes
+# of the last 4 weeks.
+class ReportReposPersonalNonOwnerPush(ReportDaily):
```

Comment:
Would it make sense to make `pushes` plural? It’s just a bit odd for me to talk about `push` in singular (also concerns the output filename `repositories-personal-non-owner-push` below and the demo data TSV file).